main.js

```diff
@@ -1,3 +1,39 @@
-import './style.css';
+const tooltipOnHoverPopup = document.getElementById("tooltip-on-hover-popup")
+const tooltipOnClickPopup = document.getElementById("tooltip-on-click-popup")
+const tooltipOnHoverButton = document.getElementById("button-tooltip-on-hover")
+const tooltipOnClickButton = document.getElementById("button-tooltip-on-click")
+const modalOnClickButton = document.getElementById("button-modal-on-click")
+const modalPopup = document.getElementById("modal-popup")
+const modalPopupOK = document.getElementById("modal-popup-ok")
+const modalPopupCancel = document.getElementById("modal-popup-cancel")
+
+const hidden = "hidden"
+const visible = "visible"
+
+tooltipOnHoverButton.addEventListener("mouseover", function whenHoverMouse() {
+    tooltipOnHoverPopup.style.visibility = visible
```

Comment:
Use `display: none;` to hide elements.
It will exclude your element from [CSSOM](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model), but visibility - not